### PR TITLE
Handle special accounts

### PR DESCRIPTION
**Pull request by AurevoirXavier**
```rs
 INFO  state_processor::system    > migrate special account `SiblId(2023)`
 INFO  state_processor::system    > migrate special account `PalletId(da/paais)`
 INFO  state_processor::system    > migrate special account `SiblId(2004)`
 INFO  state_processor::system    > migrate special account `PalletId(PotStake)`
 INFO  state_processor::system    > migrate special account `PalletId(da/ethrl)`
 INFO  state_processor::system    > migrate special account `PalletId(da/trsry)`
 INFO  state_processor::system    > migrate special account `PalletId(da/crais)`
 INFO  state_processor::system    > migrate special account `PalletId(da/ethfe)`
 INFO  state_processor::system    > migrate special account `PalletId(py/trsry)`
 INFO  state_processor::system    > migrate special account `PalletId(da/staki)`
 INFO  state_processor::system    > migrate special account `PalletId(da/s2sba)`
 INFO  state_processor::system    > migrate special account `SiblId(2000)`
 INFO  state_processor::system    > migrate special account `PalletId(da/claim)`
```

### CODE REVIEW

File: tool/state-processor/src/system/README.md [hunk 1]
```diff
@@ -34,7 +34,8 @@
   - compare the calculated one with the storage one
 
 - some remaining accounts, bridge endpoint accounts - TODO
-- special accounts - TODO
-  - parachain backing account 0x8c585F9791EE5b4B23fe82888cE576DBB69607eB
-  - bridge root account 0x726f6f7400000000000000000000000000000000
-  - pallet accounts
+- special accounts
+  - parachain backing account DONE
```
Comment (boundless-forest) [inline tool/state-processor/src/system/README.md:38]: Question: Which is the parachain backing account?

Comment (AurevoirXavier) [reply tool/state-processor/src/system/README.md:38]: #218

**Comment by boundless-forest** [comment]
> INFO  state_processor::system    > migrate special account `PalletId(py/trsry)`

Do you know what `py` means here?  A bit weird.

**Comment by AurevoirXavier** [comment]
> > INFO  state_processor::system    > migrate special account `PalletId(py/trsry)`
> 
> Do you know what `py` means here? A bit weird.

parity

**Review by boundless-forest — APPROVED** [review]
LGTM， I'll update the test later in another pr.

**Comment by AurevoirXavier** [comment]
Latest version output:
```rs
 INFO  state_processor::system    > migrate special Account(`SiblId(2023)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/paais)`)
 INFO  state_processor::system    > migrate special Account(`SiblId(2004)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(PotStake)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/ethrl)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/trsry)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/crais)`)
 INFO  state_processor::system    > migrate special Account(0x0000000000000000000000000000000000000000000000000000000000000000)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/ethfe)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(py/trsry)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/staki)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/s2sba)`)
 INFO  state_processor::system    > migrate special Account(`SiblId(2000)`)
 INFO  state_processor::system    > migrate special Account(`PalletId(da/claim)`)
```

**Review by boundless-forest — APPROVED** [review]

**Pull request merged**